presto-main/src/main/java/io/prestosql/sql/planner/planprinter/IoPlanPrinter.java

```diff
@@ -514,6 +651,26 @@ else if (writerTarget instanceof CreateReference || writerTarget instanceof Inse
             return processChildren(node, context);
         }
 
+        private EstimatedStatsAndCost getEstimatedStatsAndCost(TableScanNode node, TupleDomain<ColumnHandle> predicate, TableMetadata tableMetadata)
+        {
+            QualifiedObjectName tableName = new QualifiedObjectName(
+                    tableMetadata.getCatalogName().getCatalogName(),
+                    tableMetadata.getTable().getSchemaName(),
+                    tableMetadata.getTable().getTableName());
+            Optional<TableHandle> optTableHandle = metadata.getTableHandle(session, tableName);
```

Comment:
s/optTableHandle/tableHandle

Also `optTableHandle` looks not used, and this could result in not needed metastore call, please remove.